CMakeLists.txt

```diff
@@ -2513,6 +2513,8 @@ if(UNIX AND NOT ANDROID AND NOT APPLE)
 		DIRECTORY "${CMAKE_BINARY_DIR}/assets"
 		DESTINATION "${CMAKE_INSTALL_DATADIR}/ppsspp"
 		PATTERN ".git*" EXCLUDE
+		PATTERN "mime" EXCLUDE
+		PATTERN "lang/README.md" EXCLUDE
```

Comment:
Well, in theory I suppose someone might look at this file inside installed assets/ to consider translation, but I guess they probably wouldn't...

-[Unknown]

Reply:
Yeah, initially I thought so too. However, it is improbable that the average user is going to look into the `lang` folder and read the `README.md`. However, it works very well for people browsing GitHub. So, I have started to complete some translations where they are due. But, that’s for another PR. :wink: